contribute.md

```diff
@@ -5,21 +5,23 @@ Thank you for considering contributing to our project! We welcome contributions
 
 ## Issue Tracking  
 
-Please check our [issue tracker](https://github.com/AlabamaWaterInstitute/CloudInfra/issues) for existing issues that you can work on. If you find a new issue or have a feature request, please open a new issue so we can discuss it and prioritize it accordingly.
+Please check our [issue tracker](https://github.com/CIROH-UA/NGIAB-CloudInfra/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=) for existing issues that you can work on. If you find a new issue or have a feature request, please open a new issue so we can discuss it and prioritize it accordingly.
 
 
 ## How to Contribute
 
+1. If an Issue does not already exist, create one at https://github.com/CIROH-UA/NGIAB-CloudInfra/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=
 
-1. Fork the repository and clone it to your local machine.
-
-2. Create a new branch for your changes using the format "feature/`<feature-name>`" or "bugfix/`<bug-name>`".
+2. Click on the “Create a branch” button to create a new branch. Choose the automatically selected branch.
```

Comment:
Is anyone allowed to actually create branches on the repo?

Reply:
For us internally we should be able to create branch in this repo. But for outside colloborators, we will have to find a fix for the PR validation using forked branch.

Reply:
I can, but I'm an invited collaborator.  Also, the directions for "Create a branch" are not clear.  I checked to see if there was an obvious "Create a branch" button, and didn't see one.

Reply:
I still think that the elements in #53 may help this problem -- splitting the build and the push steps into different workflows that can be run independently (or together, if needed) will give more flexibility with testing purposed changes to the IaC components, while reserving the release (the push action which requires the dockerhub login) to be reserved for triggers such as "merged to main" which happen after review and approval and should be be able to use the internal secrets.

Reply:
> I can, but I'm an invited collaborator. Also, the directions for "Create a branch" are not clear. I checked to see if there was an obvious "Create a branch" button, and didn't see one.

Once you submit the Issue, you will see option of "Create a branch" under Development section on right side section.

Reply:
> I still think that the elements in #53 may help this problem -- splitting the build and the push steps into different workflows that can be run independently (or together, if needed) will give more flexibility with testing purposed changes to the IaC components, while reserving the release (the push action which requires the dockerhub login) to be reserved for triggers such as "merged to main" which happen after review and approval and should be be able to use the internal secrets.

Yea thank you Nels for this suggestion. @benlee0423 if you could please take a look at https://github.com/CIROH-UA/NGIAB-CloudInfra/pull/53 and try it on your end to split the build and push steps into different workflows as suggested by Nels. We can discuss once you take alook.